### ISSUE TITLE
Not attention mask is input for training

### ISSUE THREAD

**Issue by double-fire-0** [OpenBMB/MiniCPM-V#279]
From the code from [huggingface minicpmv](https://huggingface.co/openbmb/MiniCPM-Llama3-V-2_5/blob/main/modeling_minicpmv.py), I notice that the attention_mask is None when calling llama3.forward function.

It will work fine when setting batch size to 1, but it seems that is not suitable for when batch size > 1.
